src/main.css

```diff
@@ -129,6 +129,14 @@ header {
 @-moz-document url-prefix() {
     .custom-scroll {
         scrollbar-width: thin;
+        scrollbar-color: #154a82 transparent;
+    }
+
+    .dark .custom-scroll {
+        scrollbar-color: #0e6cb5 transparent;
+    }
+
+    .accent-green .custom-scroll {
         scrollbar-color: #058751 transparent;
```

Comment:
could we rely on the `var(..)` method here to refer to the colors defined in the themes? might simplify it

Reply:
fixed it by adding `rgb()` around it, seems it didn't like the hex one for some reason in firefox